multisig/proposals/arbitrumone/vip-018/commands.ts

```diff
@@ -0,0 +1,100 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId } from "src/types";
+
+import { ACM, Assets, CONVERTER_NETWORK, converters } from "./addresses";
+
+const { arbitrumone } = NETWORK_ADDRESSES;
+
+type IncentiveAndAccessibility = [number, number];
+
+interface AcceptOwnership {
+  target: string;
+  signature: string;
+  params: [];
+}
+
+interface CallPermission {
+  target: string;
+  signature: string;
+  params: [string, string, string];
+}
+
+export const grant = (target: string, signature: string, caller: string): CallPermission => {
+  const config: CallPermission = {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [target, signature, caller],
+  };
+
+  return config;
+};
+
+const incentiveAndAccessibility: IncentiveAndAccessibility = [0, 1];
```

Comment:
Let's configure the same incentives already configured on Ethereum and BNB Chain: 1e14